Components/RTShaderSystem/src/OgreShaderProgramManager.cpp

```diff
@@ -70,7 +70,9 @@ void ProgramManager::releasePrograms(const ProgramSet* programSet)
         if(prg.use_count() > ResourceGroupManager::RESOURCE_SYSTEM_NUM_REFERENCE_COUNTS + 2)
             continue;
 
```

Comment:
```suggestion
// TODO: investiage - this should not happen, but it does
if(prg.use_count() == 1)
    continue;
```
in case you dont feel like diving that deep here